index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css"
+      integrity="sha384-b6lVK+yci+bfDmaY1u0zE8YYJt0TZxLEAFyYSLHId4xoVvsrQu3INevFKo+Xir8e"
+      crossorigin="anonymous"
+    />
+    <title>Mobile-First</title>
+  </head>
+  <body>
+    <main>
+    <section class="parent">
+      <div>
+        <nav>
+          <!--container 1-->
+          <div>
+            <h5>My Logo</h5>
+          </div>
+          <!--container 2-->
+          <div>
+            <i class="bi bi-list"></i>
+          </div>
+        </nav>
+        
+        <section class="bg-img">
+          <div>
+            <h1 class="name">I'm Maris<br/>Glad to see</br> you!</h1>
+            <div class="article">
+              I’m a software developer! I can help you<br>
+              build a product , feature or
+              website Look<br>
+              through some of my work and<br>
+              experience! If you like what
+              you see and<br>
+              have a project you need coded, don’t<br>
+              hesitate to contact me.
+            </div>
+          </div>
+          <div>
+            <h3 class="connect">LET'S CONNECT</h3>
+          </div>
+          <ul>
+            <li class="icons"><a href="#"><i class="bi bi-twitter"></i></a></li>
+            
+            <li class="icons"><a href="#"><i class="bi bi-linkedin"></i></a></li>
```

Comment:
- [ ] There are some aspects you can improve on the general design to match the original design.
    - The headline content should be pushed down a little bit from the `tool bar`.
    - Use the correct `font family` used in the design and adjust the `font size`
    - Position some of your background images correctly so that they don't appear to be colliding with the headline elements
    
![image](https://user-images.githubusercontent.com/98453979/227586314-3c4870d0-3cb8-4f9e-9c8d-a10ad7ee6f7d.png)

### Checkout the original Design

![image](https://user-images.githubusercontent.com/98453979/227587572-cb770e0d-c739-4dd2-a742-ee97f6be294c.png)

Reply:
- [ ] As suggested earlier, please kindly use the correct font-family (Poppins) for all the text in your project. **Hint: You may need to import the font either to your .html file or .css file**. You can go through this [short tutorial](https://www.w3docs.com/snippets/css/how-to-import-google-fonts-in-css-file.html). :info

Reply:
As suggested earlier, please kindly use the correct `font-family` (Poppins) for all the text in your project. Hint: You may need to import the font either to your .html file or .css file. You can go through this short tutorial. ℹ️